docs/characterization/parallel_xeb.ipynb

```diff
@@ -77,6 +77,7 @@
     "circuit_library = rqcg.generate_library_of_2q_circuits(\n",
```

Comment:
<p>Make a comment what the default options are (or make all the keyword arguments explicit in this example).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/3795/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>good point. Might make sense to add factory functions / globals with common, better-named defaults like in floquet. I'll write out all the kwargs for now. </p>